src/Microsoft.Azure.WebJobs.ServiceBus/WebJobs.ServiceBus.csproj

```diff
@@ -26,4 +32,7 @@
     <ProjectReference Include="..\Microsoft.Azure.WebJobs.Host\WebJobs.Host.csproj" />
   </ItemGroup>
 
+  <Import Project="..\Microsoft.Azure.WebJobs.Shared\WebJobs.Shared.projitems" Label="Shared" />
```

Comment:
What is this project? Does this exist?

Reply:
I added that for the Sanitizer, which is internal yet shared between some projects. ServiceBus now needs it as well.

Reply:
Ah.... I was looking for this in the wrong repo :/